ws-calculator/src/main/java/org/egov/wsCalculation/controller/MeterReadingController.java

```diff
@@ -0,0 +1,60 @@
+package org.egov.wsCalculation.controller;
+
+import java.util.List;
+
+import javax.validation.Valid;
+import org.egov.wsCalculation.model.MeterConnectionRequest;
+import org.egov.wsCalculation.model.MeterReading;
+import org.egov.wsCalculation.model.MeterReadingResponse;
+import org.egov.wsCalculation.model.MeterReadingSearchCriteria;
+import org.egov.wsCalculation.model.RequestInfoWrapper;
+import org.egov.wsCalculation.service.MeterService;
+import org.egov.wsCalculation.util.ResponseInfoFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+@RestController
+@RequestMapping("/meterConnection")
```

Comment:
Recheck the endpoint name with the defined contract

Reply:
will surely change this but we need to update swagger for the same